ios-base/ViewModel/Onboarding/SignInViewModel.swift

```diff
@@ -52,7 +52,10 @@ class SignInViewModelWithCredentials {
       .login(email,
              password: password,
              success: { [weak self] in
-               self?.state = .loggedIn
+              guard let self = self else { return }
+              self.state = .loggedIn
+              AnalyticsManager.shared.identifyUser(with: self.email)
+              AnalyticsManager.shared.log(event: Event.login)
```

Comment:
Can we put just `.login`?

Reply:
Nope, since the log event expects an `AnalyticsEvent` type the compiler can't infer that Event is a valid type that easily